src/main/java/com/github/kokorin/jaffree/ffmpeg/FFmpegResultFuture.java

```diff
@@ -137,4 +138,13 @@ public boolean isCancelled() {
     public boolean isDone() {
         return resultFuture.isDone();
     }
+    
+    /**
+     * Returns a completion that can be used to chain operations after FFmpeg completes, using the
+     * {@link CompletionStage} Java 8 API 
+     * @return completion that will complete when 
```

Comment:
Is return result description unfinished?

Reply:
yes. I'll fix.